Nanoleaf-Api-Test/Program.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using System;
+using Octokit;
```

Comment:
Ocotkit is no longer needed in this project right?

Reply:
Removed from Api and Api.Test